### PR TITLE
[Keymap] plattfot - Kyria layout

### PR DESCRIPTION
**Pull request by plattfot**
## Description

Keymap for the [Kyria keyboard](https://splitkb.com/) with programming, writing in both English and Swedish and easy navigate a tiling window manager in mind.

See README.md for more info.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kyria/keymaps/plattfot/README.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
+# plattfots Keymap for [Kyria](https://github.com/splitkb/kyria)
```
Comment (fauxpark) [inline keyboards/kyria/keymaps/plattfot/README.md:1]: Please rename this file to lowercase.

File: keyboards/kyria/keymaps/plattfot/keymap.c [hunk 2]
```diff
@@ -0,0 +1,438 @@
...
+                double_tap_it = !double_tap_it;
+                return false;
+            case KC_LEAD:
+                double_tap_it = false;
+                return true;
+        }
+    } else if (double_tap_it && keycode != DBL_TAP) {
+        double_tap_it = false;
+        unregister_code16(keycode);
+        register_code16(keycode);
```
Comment (fauxpark) [inline keyboards/kyria/keymaps/plattfot/keymap.c:212]:
```suggestion
        tap_code16(keycode);
```

File: keyboards/kyria/keymaps/plattfot/keymap.c [hunk 3]
```diff
@@ -0,0 +1,438 @@
...
+            break;
+        case _ADJUST:
+            oled_write_P(PSTR("adjust\n"), false);
+            break;
+        default:
+            oled_write_P(PSTR("undefined\n"), false);
+    }
+
+    // Host Keyboard LED Status and Double tap
+    uint8_t led_usb_state = host_keyboard_leds();
```
Comment (fauxpark) [inline keyboards/kyria/keymaps/plattfot/keymap.c:278]:
```suggestion
    led_t led_state = host_keyboard_led_state();
```

File: keyboards/kyria/keymaps/plattfot/keymap.c [hunk 4]
```diff
@@ -0,0 +1,438 @@
...
+    if (IS_LED_ON(led_usb_state, USB_LED_NUM_LOCK)) {
```
Comment (fauxpark) [inline keyboards/kyria/keymaps/plattfot/keymap.c:279]:
```suggestion
    if (led_state.num_lock) {
```

File: keyboards/kyria/keymaps/plattfot/keymap.c [hunk 5]
```diff
@@ -0,0 +1,438 @@
...
+        oled_write_P(PSTR("NUMLCK "), false);
+    } else if (double_tap_it) {
+        oled_write_P(PSTR("Double "), false);
+    } else {
+        oled_write_P(PSTR("       "), false);
+    }
+
+    if (IS_LED_ON(led_usb_state, USB_LED_CAPS_LOCK)) {
```
Comment (fauxpark) [inline keyboards/kyria/keymaps/plattfot/keymap.c:287]:
```suggestion
    if (led_state.caps_lock) {
```

File: keyboards/kyria/keymaps/plattfot/keymap.c [hunk 6]
```diff
@@ -0,0 +1,438 @@
...
+        oled_write_P(PSTR("CAPLCK "), false);
+    } else if (double_tap_it) {
+        oled_write_P(PSTR("Tap    "), false);
+    } else {
+        oled_write_P(PSTR("       "), false);
+    }
+
+    oled_write_P(IS_LED_ON(led_usb_state, USB_LED_SCROLL_LOCK) ? PSTR("SCRLCK ") : PSTR("       "), false);
```
Comment (fauxpark) [inline keyboards/kyria/keymaps/plattfot/keymap.c:295]:
```suggestion
    oled_write_P(led_state.scroll_lock ? PSTR("SCRLCK ") : PSTR("       "), false);
```

File: keyboards/kyria/keymaps/plattfot/keymap.c [hunk 7]
```diff
@@ -0,0 +1,438 @@
...
+    } else {
+        render_kyria_logo();
+    }
+}
+#endif
+
+#ifdef ENCODER_ENABLE
+void encoder_update_user(uint8_t index, bool clockwise) {
+    if (index == 0) {
+        switch (biton32(layer_state)) {
```
Comment (fauxpark) [inline keyboards/kyria/keymaps/plattfot/keymap.c:310]:
```suggestion
        switch (get_highest_layer(layer_state)) {
```

File: keyboards/kyria/keymaps/plattfot/keymap.c [hunk 8]
```diff
@@ -0,0 +1,438 @@
...
+                    tap_code(KC_VOLU);
+                    clear_oneshot_layer_state(ONESHOT_OTHER_KEY_PRESSED);
+                } else {
+                    tap_code(KC_VOLD);
+                    clear_oneshot_layer_state(ONESHOT_OTHER_KEY_PRESSED);
+                }
+                break;
+        }
+    } else if (index == 1) {
+        switch (biton32(layer_state)) {
```
Comment (fauxpark) [inline keyboards/kyria/keymaps/plattfot/keymap.c:333]:
```suggestion
        switch (get_highest_layer(layer_state)) {
```

File: keyboards/kyria/keymaps/plattfot/README.md (hunk 1 above)
Comment (plattfot) [reply keyboards/kyria/keymaps/plattfot/README.md]: Done

File: keyboards/kyria/keymaps/plattfot/keymap.c [hunk 9]
```diff
@@ -0,0 +1,437 @@
...
+    // New keys
+    DBL_TAP,  // Repeat next key
+};
+
+// Double Tap feature based on post from drashna
+// https://www.reddit.com/r/olkb/comments/citkbx/double_key_press_modifier_qmkwould_work_like/ev9cue8/
+static bool double_tap_it = false;
+
+enum layers {
+    _DEFAULT = 0,
```
Comment (drashna) [inline keyboards/kyria/keymaps/plattfot/keymap.c:36]: This is more of a nit pick, but this starts off as 0, no need to explicitly set it.

```suggestion
    _DEFAULT,
```

File: keyboards/kyria/keymaps/plattfot/README.md [hunk 10]
```diff
@@ -0,0 +1,162 @@
+# plattfot's keymap for [Kyria](https://github.com/splitkb/kyria)
+
+![Keyboard](https://i.imgur.com/Pvsm973.jpg)
```
Comment (noroadsleft) [inline keyboards/kyria/keymaps/plattfot/README.md:3]: Swaps the image for a smaller version.

```suggestion
![Keyboard](https://i.imgur.com/Pvsm973l.jpg)
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**